inst/qml/doeFactorial.qml

```diff
@@ -256,6 +258,40 @@ Form
 				label:							qsTr("General full factorial design")
 				checked:						true
 			}
+
+            RadioButton
+            {
+                id:                             factorialPlackettBurman
+                name:                           "factorialPlackettBurman"
+                label:                          qsTr("Plackett-Burman design")
+
+                DropDown
+                {
+                    name: 				"PBruns"
+                    label:              qsTr("Number of runs")
+                    visible:            factorialPlackettBurman.checked
+                    indexDefaultValue: 	0
+                    values:
+                    [
+                        { value: "4",  label: qsTr("4") 	},
+                        { value: "8",  label: qsTr("8") 	},
+                        { value: "12", label: qsTr("12") 	},
+                        { value: "16", label: qsTr("16") 	},
+                        { value: "20", label: qsTr("20") 	},
+                        { value: "24", label: qsTr("24")	},
+                        { value: "28", label: qsTr("28")	},
+                        { value: "32", label: qsTr("32")	},
+                        { value: "36", label: qsTr("36")	},
+                        { value: "40", label: qsTr("40")	},
+                        { value: "44", label: qsTr("44")	},
+                        { value: "48", label: qsTr("48")	},
+                        { value: "52", label: qsTr("52")	},
+                        { value: "56", label: qsTr("56")	},
+                        { value: "60", label: qsTr("60")	},
+                        { value: "64", label: qsTr("64")	}
+                    ]
```

Comment:
any reason why people don't want 68 or so? if yes, then this could be a DoubleField with a custom validator (e.g., it must be a multiple of 4).